contrib/ossec-testing/tests/cisco_cx.ini

```diff
@@ -0,0 +1,15 @@
+[cisco cx: normal flow]
+log 1 pass = 1 2015-01-28T22:26:34.371Z 1.1.1.23 CiscoNGFW 2827 5 [ngfwEvent@9 Http_Response_Status="200" Flow_Dst_Service="tcp/80" Flow_Bytes_Sent="519" Flow_Bytes_Received="352" Event_Type="0" Count="1" Flow_DstIp="54.241.20.38" Flow_SrcIp="1.2.3.32" Url_Category_Name="Business and Industry" Flow_Bytes="871" Web_Reputation_Threat_Type="" Avc_Tag_Name="" Ev_SrcLabel="CX-CX" Response_Magic_Type="image/gif" Event_Type_Name="HTTP Complete" User_Realm="1.2.3.32" Policy_Name="Implicit Allow" Flow_Transaction_Id="4" Url="http://nexus.ensighten.com/error/e.gif?msg=Unable%20to%20get%20property%20'onmousedown'%20of%20undefined%20or%20null%20reference&lnn=-1&fn=&cid=1017&client=MSFTsupport&publishPath=prod&rid=602489&did=255875&errorName=TypeError" Identity_Source_Name="None" Auth_Policy_Name="Default" Flow_SrcIfc="inside" Flow_ConnId="28362166" Flow_DstHostName="nexus.ensighten.com" Flow_Transaction_Count="1" Ev_Id="1276022" Web_Reputation_Score="0.5" Event_Type_Action="Info" Ev_GenTime="1422484014848" Flow_DstPort=
```

Comment:
Grrrrrrr (shakes fist in air at Cisco). 

Ok weird but this could be why it's not work but working live.  I am assuming you have confirmed it works with real data?